lib/emmet-jsx-css-modules.js

```diff
@@ -58,7 +58,7 @@ export default {
 
       filters.add('jsx-css-modules', (tree) => {
         tree.children.forEach((item) => {
-          item.start = item.start.replace(/className="(.*?)"/, replFn);
+          item.start = item.start.replace(/className=[",'](.*?)[",']/, replFn);
```

Comment:
The regex character set `[",']` will match any of those three characters: `"`, `,`, or `'`. You probably meant that to be `["']`.

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions for more info.